model/src/w3src4md.F90

```diff
@@ -832,36 +849,115 @@ SUBROUTINE W3SIN4 (A, CG, K, U, USTAR, DRAT, AS, USDIR, Z0, CD,    &
       COSWIND=(ECOS(IS)*COSU+ESIN(IS)*SINU)
       TEMP=TEMP+A(IS)*(MAX(COSWIND,0.))**3
     END DO
+    !
+    LEVTAIL0= CONST0*TEMP  ! LEVTAIL is sum over theta of A(k,theta)*cos^3(theta-wind)*DTH*SIG^5/(g^2*2pi)*2*pi*SIG/CG
+                           !  which is the same as sum of E(f,theta)*cos^3(theta-wind)*DTH*SIG^5/(g^2*2pi)
+                           ! reminder:  sum of E(f,theta)*DTH*SIG^5/(g^2*2pi) is 2*k^3*E(k)
+!
+! Computation of stress supported by tail: uses table if SINTAILPAR(1)=1 , correspoding to SINTABLE = 1
```

Comment:
@pieterbartsmit These are the most recent changes to the computation of ustar, if you want to take a look.